### ISSUE TITLE
RP2040 Support

### ISSUE THREAD

**Issue by nathanbern** [waspinator/AccelStepper#32]
Does this library support the RP2040? I could not find anything in the code that suggests it doesn't, but would prefer to have a solid answer.

Thanks

**Comment by Poofjunior** [comment]
Yes, this is possible. You would need to redefine your own "Arduino.h" file, implement the missing functions with an RP2040-related implementation, and then point AccelStepper to it during compilation.

I have an in-progress project built with the pico-sdk that does just that. (Compiles. Still in progress.) Feel free to reference the folder structure [here](https://github.com/AllenNeuralDynamics/harp.pico.multistage/tree/main). The most valueable bits for adapting to your project are probably the [CMakeLists.txt](https://github.com/AllenNeuralDynamics/harp.pico.multistage/blob/main/firmware/CMakeLists.txt) and the [Arduino.h](https://github.com/AllenNeuralDynamics/harp.pico.multistage/blob/main/firmware/inc/Arduino.h) file.

Cheers--and happy hacking.
